features/marker.yml

```
@@ -0,0 +1,8 @@
+name: ::marker
+description: The `::marker` CSS pseudo-element styles the font, text, and color of a list item marker.
```

For selectors and combinators, I think it's really nice to include "select" somewhere in the description text:

```suggestion
description: The `::marker` CSS pseudo-element selects the font, text, and color of a list item marker.
```